doc/specs/bookmarks.md

```diff
@@ -0,0 +1,595 @@
+# Bookmarks
+
+## Metadata
+
+Author: killasnake0-wcd
+
+
+## Table of contents
+
+1. Motivation
+1. Business rules
+1. Use cases
+
+## Motivation
+
+Provide a system for users to bookmark torrent groups, artists, collages, 
+or requests made on the website
+
+## Business rules
+
+* Users are able to bookmark a torrent group, artist, collage, or request.
+* Users are able to remove bookmarks
+* Users are able to sort bookmarks in ascending or descending order based on different fields.
+* Users can receive notifications about certain bookmarks. 
+
+## Use cases
+
+### 1. Torrent Group Bookmarking
+
+#### 1.0 Viewing All Bookmarked Torrent Groups
+
+**Primary Actor:** Any user.
+
+**Brief:** Allows the user to view their torrent group bookmarks.
+
+**Trigger:** The user wants to view torrent group bookmarks.
+
+**Preconditions:** The user has already bookmarked at least one torrent group. 
+
+**Basic Flow:**
+ 1. The user navigates to the Bookmarks page.
+ 2. The user clicks on the Torrents hyperlink.
```

Comment:
Based on thom's suggestion below, mentioning "hyperlink" is too specific. It could be a button.
Better to say "selects torrents" or "navigates to bookmarked torrents" or something